base/models.py

```diff
@@ -140,7 +162,11 @@ def firma(self, firmatario, concedi=True, modulo=None, motivo=None):
             raise ValueError("Il modulo richiesto per l'accettazione non e' stato completato correttamente.")
 
         self.concessa = concedi
-        self.firmatario = firmatario
+        if auto:
+            # TODO: decidere indirizzo email firmatario
+            self.firmatario = firmatario #settings.DEFAULT_FROM_EMAIL
```

Comment:
Il firmatario dovra' poter essere `None` se e solo se l'azione intrapresa e' stata di natura automatica. Bisogna accertarsi che questo sia gestito bene in tutti i template esistenti o casi.